dist/pythonlibs/riotctrl_shell/tests/test_gnrc.py

```diff
@@ -34,6 +34,24 @@ def test_ping6_parser_success():
     assert "avg" in ping_res["rtts"]
 
 
+def test_ping6_parser_success2():
+    parser = riotctrl_shell.gnrc.GNRCICMPv6EchoParser()
+    ping_res = parser.parse("""
+12 bytes from ::1: icmp_seq=0 ttl=64
+12 bytes from ::1: icmp_seq=1 ttl=64
+12 bytes from ::1: icmp_seq=1 ttl=64 (DUP)
+
+--- ::1 PING statistics ---
+2 packets transmitted, 3 packets received, 1 duplicates, 0% packet loss
+round-trip min/avg/max = 0.432/0.433/0.435 ms""")
+    assert ping_res is not None
```

Comment:
With the previous comment, here it could just be (means: there's something in the dict result):
```suggestion
    assert ping_res
```